src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/ReleaseDateServiceTest.kt

```diff
@@ -206,6 +207,27 @@ class ReleaseDateServiceTest {
     assertFalse(service.isLateAllocationWarningRequired(null))
   }
 
+  @Test
+  fun `should return cut-off date as 2018-03-14 when job execution date is 2018-03-12`() {
```

Comment:
be good to name these tests after the thing you are testing - I can see you are testing 3 different dates but what do the dates mean? presumably some of them have special interactions with bank holidays

Reply:
updated the names.